spec/locale.html

```diff
@@ -68,33 +69,33 @@ <h1>Intl.Locale ( _tag_ [ , _options_ ] )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-apply-options-to-tag" aoid=ApplyOptionsToTag>
-      <h1>ApplyOptionsToTag ( _tag_, _options_ )</h1>
-
+    <emu-clause id="sec-updatelanguageid" type="abstract operation" oldids="sec-apply-options-to-tag">
+      <h1>
+        UpdateLanguageId (
```

Comment:
I'm a big fan of getting LanguageId into the name of this AO, given that the `unicode_language_id` is what it works with